src/content/developers/docs/scaling/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Scaling
+description: An introduction to the different scaling options currently being developed by the Ethereum community.
+lang: en
+sidebar: true
+incomplete: true
```

Comment:
```suggestion
```

Reply:
I think we can mark this page as "complete". It's pretty well fleshed out. I'm good with the subpages being "incomplete".